src/main/webapp/script.js

```diff
@@ -36,26 +36,5 @@ window.onload = function () {
         { theme: "outline", size: "large" }  // customization attributes
     );
     google.accounts.id.prompt(); // also display the One Tap dialog
-
-async function loadPosts() {
```

Comment:
Instead of deleting all of this, can you reuse it and adapt it for your endpoint.

Perhaps call it inside getPosts(), where this specific function will load unreplied posts, and then you can create another function for getUnrepliedPosts.

So, rename this and reuse it instead of deleting it.

Reply:
I think I do need this because I have includes both features (get the replied post and unreplied post) in my getPost function, which has been tested completely, and I will separate that function.

Reply:
Sure. I don't think it's a bad idea to separate the javascript functions into this file, therefore being able to reuse this function. However, not too critical to do so for now.